kubernetes-csi-external-provisioner.yaml

```
@@ -0,0 +1,37 @@
+package:
+  epoch: 0
+  name: kubernetes-csi-external-provisioner
+  version: 3.5.0
+  description: A dynamic provisioner for Kubernetes CSI plugins.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - build-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes-csi/external-provisioner
+      expected-commit: ab68435fa2386edf8577a02b5bf2dc4baeadbf57
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      packages: ./cmd/csi-provisioner
+      output: csi-provisioner
+      ldflags: -X main.version=${{package.version}}
```

```suggestion
      ldflags: -X main.version=${{package.version}}
```

nit: the Makefile's for `kubernetes-csi/*` also throw in https://github.com/kubernetes-csi/external-provisioner/blob/master/release-tools/build.make#LL81C33-L81C33, but I think they're redundant here?